app/helpers/distribution_helper.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module DistributionHelper
+  VALUE = '_'
+
+  # Consider these as private methods that are (and can only be) tested in their own
+  # context, hence there is no specific test suite for this file.
+  def usable_questions
+    @questionnaire_content = questionnaire.content
+    range_questions(@questionnaire_content) + other_questions(@questionnaire_content)
+  end
+
+  def range_questions(questionnaire_content)
+    rg_instance = RangeGenerator.new
+    questionnaire_content
+      .select { |question| question[:type] == :range }
+      .map do |question|
+      rg_instance.send(:range_slider_minmax, question).merge(id: question[:id].to_s,
+                                                             type: question[:type],
+                                                             combines_with: question[:combines_with],
```

Comment:
perhaps nest the 'combines_with' in a `:score` block.

Reply:
I think the score block will be something different as we discussed. This is purely for combining histograms.